drivers/SmartThings/matter-sensor/profiles/air-quality-sensor-common.yml

```diff
@@ -0,0 +1,20 @@
+name: air-quality-sensor-common
```

Comment:
Since some of these air quality sensor profile names are getting long, how about we abbreviate `air-quality-sensor` to `aqs` to make them all shorter?

Reply:
Don ein [073a879](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1422/commits/073a879332cde2dc827046b5d50efc804313e45e)!